app/vite.config.js

```diff
@@ -136,12 +137,16 @@ function directusExtensions() {
 	];
 
 	async function loadExtensions() {
-		await ensureExtensionDirs(EXTENSIONS_PATH, NESTED_EXTENSION_TYPES);
-		const packageExtensions = await getPackageExtensions(API_PATH, APP_OR_HYBRID_EXTENSION_PACKAGE_TYPES);
-		const localPackageExtensions = await resolvePackageExtensions(EXTENSIONS_PATH);
-		const localExtensions = await getLocalExtensions(EXTENSIONS_PATH, APP_OR_HYBRID_EXTENSION_TYPES);
+		const localExtensions = extensionsPathExists ? await resolveExtensions(EXTENSIONS_PATH) : [];
+		const dependencyExtensions = await resolveDependencyExtensions(API_PATH);
 
-		const extensions = [...packageExtensions, ...localPackageExtensions, ...localExtensions];
+		/*
+		 * @TODO
+		 * These aren't deduplicated, whereas the production ones are. This has seemingly
+		 * always been the case. Is this a bug?
+		 * @see /api/src/extensions/lib/get-extensions.ts
+		 */
+		const extensions = [...localExtensions, ...dependencyExtensions];
```

Comment:
Would it make sense / be possible to share the code from `get-extensions.ts`?

Reply:
No, that's using the APIs logger as well (although.... refactoring that to @directus/logger would be nice)